pkg/tfgen/test_data/convertExamples/aws_lambda_function_out.md

```diff
@@ -53,14 +53,14 @@ import pulumi
 import pulumi_archive as archive
 import pulumi_aws as aws
 
-assume_role = aws.iam.get_policy_document(statements=[aws.iam.GetPolicyDocumentStatementArgs(
-    effect="Allow",
-    principals=[aws.iam.GetPolicyDocumentStatementPrincipalArgs(
-        type="Service",
-        identifiers=["lambda.amazonaws.com"],
-    )],
-    actions=["sts:AssumeRole"],
-)])
+assume_role = aws.iam.get_policy_document(statements=[{
```

Comment:
Will we have to update provider tests downstream once this is released?

Reply:
I suspect providers do not have golden tests for docs - we'll see the diffs in the sdks though